docs/UserGuide.md

```diff
@@ -478,69 +551,114 @@ Format: `todo add d/DESCRIPTION date/DATE pr/PRIORITY [t/TAG]…​`
+
+![Todo Add](images/TodoAdd.png)
 
 <br>
 
 #### 7.3.2. Editing a todo : `todo edit`
 
-Edits an existing todo in your SoConnect.
+You can update the information of a todo easily using the `todo edit` command as shown below. All you need is the `INDEX` of the todo you want to modify along with the parameters you want to update.
 
-Format: `todo edit INDEX [d/DESCRIPTION] [date/DATE] [pr/PRIORITY] [t/TAG]…​`
+**Format:** `todo edit INDEX [d/DESCRIPTION] [date/DATE] [pr/PRIORITY] [t/TAG]…​`
 
-* Edits the todo at the specified [`INDEX`](#).
-* At least one of the optional fields must be provided.
-* Parameters given will overwrite the existing values completely.
-  * For example, giving 1 or more tag(s) in the edit command will replace all existing tags with the ones given in the edit command.
-  * `Coming soon in v1.5`, you can use `tag add` and `tag remove` to modify tags in a todo instead of only using `todo edit`.
+* At least 1 of the 4 optional fields must be provided.
+* Existing information will be overwritten with the parameters provided. Information of the parameters not provided will remain unchanged.
 
-Examples:
-*  `todo edit 1 d/Read notes for ST2334` Edits the description of the 1st todo to be `Read notes for ST2334`.
-*  `todo edit 1 pr/medium t/ST2334` Edits the priority of the 2nd contact to be `medium` and changes its tags to just `ST2334`.
 
-<br>
+<div markdown="block" class="alert alert-primary">
+**:memo: Note:**<br/>
+[Tags](#tag) have to be created first before you can add them to a contact.
 
-#### 7.3.3. Deleting a todo : `todo delete`
+* Refer to [`Creating a Tag`](#741-creating-a-tag-tag-create) on how to create a tag.
+</div>
 
-Deletes the specified todo from your SoConnect.
+<div markdown="block" class="alert alert-info">
+**:information_source: info:** <br/>
 
-Format: `delete INDEX`
+* You can use `todo edit` command to add or remove tags to a todo.
+  * To add a tag, you can include all the existing tags in the todo as parameter together with the new tags that you wish to add.
+  * To remove a tag, you can include all the existing tags in the todo excluding the tags that you wish to remove.
+* `Coming soon in v1.5`, you can use `tag add` and `tag remove` to modify tags in a todo instead of only using `todo edit` to type all the tags in the todo to make modifications.
 
-* Deletes the todo at the specified `INDEX`.
+</div>
 
-Examples:
-* `todo show` followed by `todo delete 2` deletes the 2nd todo shown in your SoConnect.
+**Example Input in Command Box:**
+
+```
+`todo edit 2 pr/medium t/cs2103 t/cs2100`
+```
+
+**Example Result:**
+
+Before:
+
+![Todo edit before](images/TodoEditBefore.png)
+
+After:
+
+![Todo edit after](images/TodoEditAfter.png)
+
+<br>
+
+#### 7.3.3. Deleting a todo : `todo delete`
+
+You can delete a particular todo after you have completed the todo or you no longer wish to keep it. This can be done easily using the `todo delete` command. All you need is the `INDEX` of the contact!
```

Comment:
```suggestion
You can delete a particular todo after you have completed the todo or you no longer wish to keep it. This can be done easily using the `todo delete` command. All you need is the `INDEX` of the todo!
```